package.json

```diff
@@ -121,5 +121,10 @@
     "src/**/*.{js,jsx,ts,tsx}": [
       "pretty-quick --staged"
     ]
+  },
+  "eslintConfig": {
+    "extends": [
+      "plugin:storybook/recommended"
+    ]
   }
```

Comment:
What if we add it in the eslint config file instead of adding it in the package.json?

Reply:
automatic by storybook cli, will change